### PR TITLE
[Docs] Add cloud default values for query complexity

### PR DESCRIPTION
**Pull request by justindeguzman**
### Changelog category (leave one):
- Documentation (changelog entry is not required)

### Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):
Add default cloud values for query complexity

### CODE REVIEW

**Comment by robot-clickhouse-ci-1** [comment]
*This is an automated comment for commit 32305adb0865f67f589565f8e2416dcdf991c693 with description of existing statuses. It's updated for the latest CI running*

[⏳ Click here](https://s3.amazonaws.com/clickhouse-test-reports/59784/32305adb0865f67f589565f8e2416dcdf991c693/ci_running.html) to open a full report in a separate page

<table>
<thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody>
<tr><td>CI running</td><td>A meta-check that indicates the running CI. Normally, it's in <b>success</b> or <b>pending</b> state. The failed status indicates some problems with the PR</td><td><a href="https://s3.amazonaws.com/clickhouse-test-reports/59784/32305adb0865f67f589565f8e2416dcdf991c693/ci_running.html">⏳ pending</a></td></tr>
<tbody>
</table>

File: docs/en/operations/settings/query-complexity.md [hunk 1]
```diff
@@ -28,6 +28,8 @@ The maximum amount of RAM to use for running a query on a single server.
 
 The default setting is unlimited (set to `0`).
 
+Cloud default value: 17 GB.
```
Comment (alexey-milovidov) [inline docs/en/operations/settings/query-complexity.md:31]:
```suggestion
Cloud default value: half the RAM amount of the replica.
```

File: docs/en/operations/settings/query-complexity.md [hunk 2]
```diff
@@ -104,7 +106,9 @@ Possible values:
 - Maximum volume of RAM (in bytes) that can be used by the single [GROUP BY](../../sql-reference/statements/select/group-by.md#select-group-by-clause) operation.
 - 0 — `GROUP BY` in external memory disabled.
 
-Default value: 0.
+Default value: `0`.
+
+Cloud default value: 8 GB.
```
Comment (alexey-milovidov) [inline docs/en/operations/settings/query-complexity.md:111]:
```suggestion
Cloud default value: half the memory amount per replica.
```

File: docs/en/operations/settings/query-complexity.md [hunk 3]
```diff
@@ -115,6 +119,8 @@ Enables or disables execution of `ORDER BY` clauses in external memory. See [ORD
 
 Default value: 0.
 
+Cloud default value: 8 GB.
```
Comment (alexey-milovidov) [inline docs/en/operations/settings/query-complexity.md:122]:
```suggestion
Cloud default value: half the memory amount per replica.
```

File: docs/en/operations/settings/query-complexity.md [hunk 4]
```diff
@@ -129,18 +135,26 @@ What to do if the number of rows received before sorting exceeds one of the limi
 
 ## max_result_rows {#setting-max_result_rows}
 
-Limit on the number of rows in the result. Also checked for subqueries, and on remote servers when running parts of a distributed query.
+Limit on the number of rows in the result. Also checked for subqueries, and on remote servers when running parts of a distributed query. No limit is applied when value is `0`.
+
+Default value: `0`.
+
+Cloud default value: `500000`.
```
Comment (alexey-milovidov) [inline docs/en/operations/settings/query-complexity.md:142]:
```suggestion
Cloud default value: `0`.
```

Comment (alexey-milovidov) [reply docs/en/operations/settings/query-complexity.md:142]: @justindeguzman, you confused it with the value set by the SQL console.

File: docs/en/operations/settings/query-complexity.md [hunk 5]
```diff
@@ -129,18 +135,26 @@ What to do if the number of rows received before sorting exceeds one of the limi
...
 ## result_overflow_mode {#result-overflow-mode}
 
-What to do if the volume of the result exceeds one of the limits: ‘throw’ or ‘break’. By default, throw.
+What to do if the volume of the result exceeds one of the limits: ‘throw’ or ‘break’.
 
 Using ‘break’ is similar to using LIMIT. `Break` interrupts execution only at the block level. This means that amount of returned rows is greater than [max_result_rows](#setting-max_result_rows), multiple of [max_block_size](../../operations/settings/settings.md#setting-max_block_size) and depends on [max_threads](../../operations/settings/settings.md#max_threads).
 
+Default value: `throw`.
+
+Cloud default value: `break`.
```
Comment (alexey-milovidov) [inline docs/en/operations/settings/query-complexity.md:156]:
```suggestion
Cloud default value: `throw`.
```

Comment (alexey-milovidov) [reply docs/en/operations/settings/query-complexity.md:156]: @justindeguzman, you confused it with the value set by the SQL console.

**Pull request merged**